src/scss/includes/components/poiItem.scss

```diff
@@ -0,0 +1,33 @@
+.poiItem {
+  padding: 16px 30px;
```

Comment:
According to the designs, `30px` is too much. You can see the content should start and stop at the same level as the separator bars. `18px` seems to be ok.

Reply:
I removes the extra space taken by the "red border" when hovering on the left. 4px was takend even when not hovering. It seems ok to me, what do you think ?

Reply:
Yep, thank you. Now content and separator are correctly aligned again. (I still think personnaly we lose too much space with this large padding, but this is consistent with the design so :man_shrugging:)